Lesson-3.js

```diff
@@ -0,0 +1,5 @@
+//Lesson-3
+
+console.log('Robert')
+console.log('Stannis')
+console.log('Renly')
```

Comment:
Remember to add a new line at the end of the file

Reply:
Thanks! I will remember it.